tests/foreman/api/test_docker.py

```diff
@@ -1346,10 +1343,25 @@ def test_positive_create_with_external_registry(self):
         @Assert: The docker container is created and the image is pulled from
         the external registry
 
-        @caseautomation: notautomated
-
         @CaseLevel: Integration
         """
+        repo_name = 'rhel'
```

Comment:
any repo from our external registry. as it's red hat one, no busybox is in there for obvious reasons, and frankly i'm not aware which repo is pretty stable to use, but i believe that `rhel` should be there for sure, so we can safely use it :)